DEVELOPMENT.md

```diff
@@ -85,13 +99,34 @@ Follow the steps below to setup your development environment, replacing `<sgx|vi
    Note this command should not be used for a production instance, as it will leave the service
    open to all.
 
-### Development Docker image
+## Configuring
 
-For more details refer to [docker/README.md](./docker/README.md).
+The application expects the configuration to be submitted via the CCF proposals, for that you could use the CLI.
 
-1. Build the image first `PLATFORM="virtual" ./docker/build.sh`
+```sh
+echo <<< EOL
+{
+    "policy": {
+        "policy_script": "export function apply(profile, phdr) { if (!phdr.issuer) {return 'Issuer not found'}; const iss=phdr.issuer.split(':eku:'); if (iss.length !== 2 || iss[1] !== '1.3.6.1.4.1.311.10.3.13') { return 'Invalid EKU'; } }"
```

Comment:
Enforcing an EKU without enforcing a root hash isn't very meaningful: anyone could issue a cert with that EKU, I suggest adding a check on the root as well.

Reply:
instead updated 2 policy_script examples to check the profile and issuer. The issuer check compares to the full issuer string now